src/elements/menu/menu-button/menu-button.visual.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { html, nothing, type TemplateResult } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+
+import type { visualRegressionFixture } from '../../core/testing/private.js';
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualDiffFocus,
+  visualDiffHover,
+} from '../../core/testing/private.js';
+
+import './menu-button.js';
+
+describe(`sbb-menu-button`, () => {
+  const defaultArgs = {
+    amount: 123 as number | undefined,
+    iconName: 'tick-small',
+    label: 'Button',
+    disabled: false,
+    slottedIcon: false,
+  };
+
+  const template = ({
+    amount,
+    iconName,
+    label,
+    disabled,
+    slottedIcon,
+  }: typeof defaultArgs): TemplateResult => html`
+    ${repeat(
+      new Array(3),
+      (_, index) => html`
+        <sbb-menu-button
+          amount=${amount || nothing}
+          icon-name=${iconName || nothing}
+          .disabled=${disabled}
```

Comment:
doesn't matter but would also work :P
```suggestion
          ?disabled=${disabled}
```